### PR TITLE
Simple Python IDE using Tkinter

### LINKED ISSUE #764: Simple  Python IDE using Tkinter
# Aim

To make a Simple  Python IDE using Tkinter with a GUI.

# Details
In this project, I will deploy a simple  Python IDE application with the help of the Tkinter module. This application just like a python IDE. we can save our code and run this IDE. we can open our exciting Project in this IDE.

@avinashkranjan I am a GSSoC'21 participant and I think this is an interesting python script of the GUI domain. Kindly allow me to work on this issue and contribute to this project!

### PR DESCRIPTION
**Pull request by Priyadarshan2000**
# Description

I have made a Simple Python IDE using the Tkinter module in Python. This refers to issue number #764 which was assigned to me. No dependencies are required for the change.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: Simple Python IDE using Tkinter/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Simple Python IDE using Tkinter
+This is a Simple Python IDE using Tkinter. We can Run our Python Code here. we open our saved code in this IDE. 
+when you can't save your code and run it an error message show **"Please save your code"**. 
+This Python IDE develops using Tkintertable & subprocess. run module. 
+After your code will run successfully, to exit from the IDE go to File and click the Exit Button.
+
+
+# Setup instructions
```
Comment (kaustubhgupta) [inline Simple Python IDE using Tkinter/README.md:8]: level two (##)

File: Simple Python IDE using Tkinter/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+There are two ways to run it on your Linux,MAC or Windows
+
+- Run the executable(.exe) file.
+- Run the program in your IDE or terminal/command line.
+
+### Requirements:
```
Comment (kaustubhgupta) [inline Simple Python IDE using Tkinter/README.md:14]:
```suggestion
## Requirements:
```

File: Simple Python IDE using Tkinter/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+```bash
+pip install subprocess.run
+```
+Click on the **Click Here** to see the Simple Python IDE using Tkinter Demo on YouTube.
+
+| Name of Script | YouTube Link |  Author | Tools Used |
+| --- | --- | --- | --- 
+|Simple Python IDE using Tkinter| [Click Here](https://www.youtube.com/watch?v=-oGVdnelHv8)| [Priyadarshan Ghosh](https://github.com/Priyadarshan2000) | tkintertable, subprocess.run
+
+# Output
```
Comment (kaustubhgupta) [inline Simple Python IDE using Tkinter/README.md:29]: level two (##)

File: Simple Python IDE using Tkinter/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+
+![image](https://user-images.githubusercontent.com/62868878/112800376-32ae3c00-908d-11eb-96ac-38b7d1f2fb76.png)
+# Demo
```
Comment (kaustubhgupta) [inline Simple Python IDE using Tkinter/README.md:33]: give space before this line and level two (##)

File: Simple Python IDE using Tkinter/README.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+![20210329_130409](https://user-images.githubusercontent.com/62868878/112802134-7013c900-908f-11eb-806a-157f8b6a6770.gif)
+
+
+# Author(s)
```
Comment (kaustubhgupta) [inline Simple Python IDE using Tkinter/README.md:37]: level two (##)

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Code is working. Just minor changes in readme:

**Comment by Priyadarshan2000** [comment]
@kaustubhgupta sir what change in the readme file.

**Comment by Priyadarshan2000** [comment]
@kaustubhgupta sir I change my Readme file. please check this.

**Review by kaustubhgupta — APPROVED** [review]

**Review by Kushal997-das — APPROVED** [review]
LGTM 💯

**Review by santushtisharma10 — APPROVED** [review]
@Priyadarshan2000 💯💯

**Review by antrikshmisri — APPROVED** [review]
LGTM

**Review by avinashkranjan — APPROVED** [review]
LGTM..Great Job @Priyadarshan2000

**Comment by Priyadarshan2000** [comment]
Thanks @avinashkranjan sir

**Comment by avinashkranjan** [comment]
> Thanks @avinashkranjan sir 😊😊

Keep up the good work..😄

**Pull request merged**